src/OrchardCore.Modules/OrchardCore.ArchiveLater/Views/ArchiveLaterPart.SummaryAdmin.cshtml

```diff
@@ -0,0 +1,8 @@
+@model ArchiveLaterPartViewModel
+
+@if (Model.ScheduledArchiveUtc.HasValue)
+{
+    <div>
+        <span class="hint">@T["Scheduled to be archived on {0}", (object)(await DisplayAsync(await New.DateTime(Utc: Model.ScheduledArchiveUtc)))]</span>
```

Comment:
@Piedone I mean this hint

Reply:
Do you mean that we shouldn't display this once `ScheduledArchiveUtc` passed? That's a good idea, yes, in that case we should hide this.

Reply:
Exactly, especially after it's archived

Reply:
So if we still set the date to null when the item is archived, nothing to do here

But it depends on what we may want, see my last comment in the main thread.